frontend/document/document.lua

```diff
@@ -495,11 +495,21 @@ Draw page content to blitbuffer.
 --]]
 function Document:drawPage(target, x, y, rect, pageno, zoom, rotation, gamma, render_mode)
     local tile = self:renderPage(pageno, rect, zoom, rotation, gamma, render_mode)
-    target:blitFrom(tile.bb,
-        x, y,
-        rect.x - tile.excerpt.x,
-        rect.y - tile.excerpt.y,
-        rect.w, rect.h)
+    -- Enable SW dithering if requested (only available in koptoptions)
+    -- Much Like ReaderView, also enforce SW dithering in PicDocument if the device can't do HW dithering...
+    if (self.is_pic and CanvasContext:hasEinkScreen() and not CanvasContext:canHWDither() and CanvasContext.fb_bpp == 8) or (self.configurable.sw_dithering and self.configurable.sw_dithering == 1) then
```

Comment:
This record breaking 202 chars long line deserves to be contemplated more easily, and so shown on more than one line :)

Reply:
Heh, it did not feel as long as that over here ;p.

The curse of giant widescreens, I guess ;p.